### ISSUE TITLE
Scan the training sample size, check validation loss performance of all tasks.

### ISSUE THREAD

**Issue by jpata** [HEP-KBFI/ml-tau-en-reg#24]
- with the full set of final features (i.e. after https://github.com/HEP-KBFI/ml-tau-en-reg/issues/17)
- for all 3 models, for all 3 tasks
- rework `https://github.com/HEP-KBFI/ml-tau-en-reg/blob/main/enreg/scripts/submit-pytorch-gpu-all.sh` to scan the additional parameter `fraction_train` over e.g. values 0.1 to 0.8 in steps of 0.1 (8 steps)
- all together, this would be 3x3x8 = 72 trainings, most of them taking a few GPU hours max
- note that currently the `trainModel.py` code takes `fraction_valid` samples right after the `fraction_train` samples, therefore the validation sample will be statistically different for each `fraction_train`
